src/App.tsx

```diff
@@ -28,15 +29,21 @@ const AppContextProvider = React.lazy(
 );
 
 export const App = () => {
+  const isStorybook = !!process.env.IS_STORYBOOK_VIEW;
+
   return (
     <React.Suspense fallback={<Loading />}>
       <ChakraProvider theme={theme}>
         <Fonts />
-        <AuthWalletContextProvider>
-          <AppContextProvider>
-            <RouterProvider router={router} />
-          </AppContextProvider>
-        </AuthWalletContextProvider>
+        {isStorybook ? (
+          <RouterProvider router={storybookRouter} />
```

Comment:
As a dev, i would prefer to go to `/storybook` to view all components while developing and not run a separate command so i'd rather have the conditional logic here placed in router.tsx. Can use `NODE_ENV` to determine if the page should be included. But just a personal preference.

Reply:
i think storybook is meant for non devs instead and is maintained by devs. but we can replace this with vitebook instead overall later. what do you think?

Reply:
I think since this is open source codebase, newcomers (devs) would benefit from storybook. to see a doc of all possible props